test/integration/package_id/compatible_test.py

```diff
@@ -706,3 +709,83 @@ def validate_build(self):
             pkga = liba["packages"][0][pkg_index]
             assert pkga["info"]["compatibility_delta"] == {"settings": [["compiler.cppstd", "14"]]}
             assert pkga["build_args"] == "--requires=liba/0.1 --build=compatible:liba/0.1"
+
+
+@pytest.mark.skipif(platform.system() != "Windows", reason="MSVC combination requires Windows profile, but the basic idea works everywhere")
```

Comment:
I added this to show that the combination logic is sound, but for the docs we'll drop the msvc variability